### DISCUSSION TITLE
How to make `print()` statement output colorful?

### DISCUSSION THREAD

**Discussion by jawad5311** [borntyping/python-colorlog#119] [Support] [answered]
How can I make my `print()` statement colorful?  
Is there any key for the `print()` statement which I have to add to the `log_colors` dict to make it happen?
Any help would be appreciated. Thanks.

**Comment by borntyping** [comment, marked as answer]
`colorlog` only works with the standard Python `logging` library. You might want to look at the [colorama](https://pypi.org/project/colorama/) or [crayons](https://pypi.org/project/crayons/) packages if you want to add colors to print statements.
